package.yaml

```diff
@@ -193,6 +193,12 @@ executables:
       - string-interpolate == 0.3.*
     verbatim:
       default-language: GHC2021
+    ghc-options:
+      - -threaded
+      - -rtsopts
```

Comment:
`rtsopts` is required to be able to pass the `-A` flag? It's strange I seem to be able to set some RTS flags with the default option.

GHC makes a security warning about using this option so we need to be careful. 

https://downloads.haskell.org/ghc/latest/docs/users_guide/phases.html#ghc-flag--rtsopts[=⟨none|some|all|ignore|ignoreAll⟩]

"""
In GHC 6.12.3 and earlier, the default was to process all RTS options. However, since RTS options can be used to write logging data to arbitrary files under the security context of the running program, there is a potential security problem. For this reason, GHC 7.0.1 and later default to -rtsopts=some.
"""

Reply:
I've added a comment